apps/src/code-studio/components/libraries/LibraryPublisher.jsx

```diff
@@ -244,29 +245,24 @@ export class LibraryPublisher extends React.Component {
   };
 
   unpublish = () => {
-    const {unpublishProjectLibrary, libraryClientApi} = this.props;
-
-    unpublishProjectLibrary(
-      libraryClientApi.channelId,
-      libraryClientApi,
-      this.onUnpublishComplete
+    const {libraryClientApi, onUnpublishSuccess} = this.props;
```

Comment:
reverting this to its [original state](https://github.com/code-dot-org/code-dot-org/blob/8bf16b29ca5c8f560ad5caaa31e1419951c87cd2/apps/src/code-studio/components/libraries/LibraryPublisher.jsx#L241-L257) prior to #33374 -- the `projectsRedux#unpublishProjectLibrary` method calls the channels API on its own, which we don't want to do because we need to call `setLibraryDetails`, which also calls the channels API